packages/launchpad/src/setup/FrameworkOptionsFooter.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="flex bg-gray-50 h-65px py-12px px-17px text-gray-800 items-center">
+    <div>
+      <Icon
```

Comment:
Can you also apply some more neutral colors to this such as `gray-500` for the strokes and `gray-100` for the fills?

Another option would be `indigo-500` and `indigo-100` so that it feels more connected to the link.